debian/neo-go.prerm

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+# prerm script for neo-go
```

Comment:
And this one.

Reply:
4 files make full templates set for service lifecycle.
Never considered them useless but I'll gladly remove these 2.